### ISSUE TITLE
User experience when Torque compute cluster is down

### ISSUE THREAD

**Issue by berland** [equinor/ert#5583]
**Describe the bug**
If the compute cluster is down, the user experience is possibly suboptimal. 

ERT crashes and the user is left with the message in the terminal:
```
Failed to get torque job id from file: /tmp/enkf-submit-std-23317-981970
qsub command                      : qsub
File content:  []
torque_job_parse_qsub_stdout:
QObject::~QObject: Timers cannot be stopped from another thread
```

**To reproduce**
Steps to reproduce the behaviour:
1. Run ERT when the compute cluster is down, or find some qsub options that can be perturbed such that the qsub command will fail
2. Or mock the qsub command to one that returns an empty string.

**Expected behaviour**
A better error message.

**Environment**
 - OS: RHEL7
 - ERT/Komodo release: 2023.05.02
 - Python version: 3.8
 - Remote/HPC execution involved: yes

**Comment by berland** [comment]
Turns out this also happens if you have typo in the compute queue.

**Comment by xjules** [comment]
We should also include what happens when we use Torque on onprem nodes.

**Comment by xjules** [comment]
Current consensus is to just improve the error message from the queue, when executing the qsub command.

**Comment by berland** [comment]
This is now resolved in #5940 . 

If writing an erroneous queue name, the user experience in the terminal looks like this:
```
$ ert ensemble_experiment poly_torque.ert

    --> Running forecast for iteration: 0

    |                                                                               1/1 |                                                             |   0% Running time: 2 seconds

    ■ Waiting           4/4
    ■ Pending           0/4
    ■ Running           0/4
    ■ Failed            0/4
    ■ Finished          0/4
    ■ Unknown           0/4

    --> Running forecast for iteration: 0

    |                                                                               1/1 |                                                             |   0% Running time: 2 seconds

    ■ Waiting           4/4
    ■ Pending           0/4
    ■ Running           0/4
    ■ Failed            0/4
    ■ Finished          0/4
    ■ Unknown           0/4

Failed to get torque job id from file: /tmp/enkf-submit-std-518-155041 
qsub command: qsub 
qsub output:  
qsub errors:  qsub: Unknown queue

torque_job_parse_qsub_stdout: 
```
(after some minutes, due to the retries that are put into action whenever `qsub` fails).
